File: Pyramid/bloc.cpp
#include "bloc.h"

Bloc::Bloc()
{
}

Bloc::Bloc(float x, float y, float z, float width, float height, float depth)
{
    this->x = x;
    this->y = y;
    this->z = z;
    this->width = width;
    this->height = height;
    this->depth = depth;
}

File: Pyramidjs/public/js/Bloc.js
var Bloc = function(x, y, z, w, h , d, type, description){
    this.type = type || 0;
    this.description = description || null;
	this.x = x;
	this.y = y;
	this.z = z;
	this.width = w;
	this.height = h;
	this.depth = d;
}

Bloc.prototype.init = function(json){
	for(var i in json){
		this[i] = json[i];
	}
}
